### DISCUSSION TITLE
Correct guidance for <br> line element

### DISCUSSION THREAD

**Discussion by tamsinewing555** [mdn/mdn-community#198] [Content]
Hi, I'd like to propose an update to your guidance page on the line break element '<br>': https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Under the 'Accessibility concerns' heading, you say that "it is problematic for people who navigate with the aid of screen reading technology. Screen readers may announce the presence of the element, but not any content contained within <br>s. This can be a confusing and frustrating experience for the person using the screen reader."

My understanding and that of a colleague, who is a fully blind screen reader (SR) user, is that SRs merely pause at the end of the line and the user needs to press the down arrow to continue after each line break element.

In the few scenarios that are appropriate for using the line break element (such as poems and addresses, when there is a semantic reason for placing content on a new line), this is in fact helpful.

It helps SR users see how a poem has been structured, and it helps us understand in an address, for example, where a street name ends and a town name begins.

However, of course, if the line break element is used incorrectly, the a11y concern is that SRs may simply stop reading at a '<br>', forcing the user to command the SR to continue reading, which can be confusing and frustrating where a break in what's being read or announced by the SR is unexpected.

The SR users that I know do not experience SRs leaping over (and not announcing) the content contained in between the opening and closing tags of the line break element, as your guidance appears to indicate.

It would be great if you could clarify and correct this on your line break element guidance page as there's a link to it from the NZ Government Web Accessibility project site:
https://govtnz.github.io/web-a11y-guidance/wct/text-content/markup-for-specific-words-and-text.html.

Thank you :)

**Comment by teoli2003** [comment]
cc/ @estelle

**Comment by estelle** [comment]
The note specifies that a series of <br/> elements should not be used. In poetry, there wouldn't be multiple <br/> elements in a row; so this article does not dissuade that. If you have a specific suggestion, you can click on the "[Edit on GitHub](https://github.com/mdn/content/edit/main/files/en-us/web/html/element/br/index.md)" link at the bottom of the page and suggest an edit.
